### PR TITLE
Add format salary helper method to vacancy model

### PR DESCRIPTION
**Pull request by C-gyorfi**
## Jira ticket URL:
https://dfedigital.atlassian.net/secure/RapidBoard.jspa?rapidView=21&modal=detail&selectedIssue=TEVA-215

## Changes in this PR:
- Add `format_salary` helper method to vacancy model, to allow any correct salary formats, including comma and decimal point.
- Update hint text for salary field

### Before
Screenshot 2019-10-09 at 13 09 51

### After
Screenshot 2019-10-09 at 13 09 27

## Next steps:
- [ ] Content review

### CODE REVIEW

**Comment by dushan-madetech** [comment]
@C-gyorfi what happens if you put in 1 decimal place E.g. 25000.4? Could you also write a test for that?

**Comment by C-gyorfi** [comment]
> @C-gyorfi what happens if you put in 1 decimal place E.g. 25000.4? Could you also write a test for that?

Good shout, I've updated the regex in the validator in 8ab6118

File: spec/models/vacancy_spec.rb [hunk 1]
```diff
@@ -186,6 +186,41 @@
...
+
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '20000'
+      end
+
+      it 'can accept salary format with one decimal point' do
+        job = build(:vacancy, minimum_salary: '25000.4')
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '25000.4'
+      end
```
Comment (dushan-madetech) [inline spec/models/vacancy_spec.rb]: this seems to go against the acceptance criteria of the ticket. I think that this should be rejected based on the acceptance criteria

Comment (C-gyorfi) [reply spec/models/vacancy_spec.rb]: Oh, in that case, the behaviour was correct before this test!

File: spec/models/vacancy_spec.rb [hunk 2]
```diff
@@ -186,6 +186,45 @@
...
+
+      it 'can accept salary format with comma' do
+        job = build(:vacancy, minimum_salary: '20,000')
+
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '20000'
+      end
+
+      it 'rejects salary format with incorrect comma placement' do
+        incorrect_salary_format = ['20,,000', '200,00', ',200'].sample
```
Comment (Renny4Real) [inline spec/models/vacancy_spec.rb]: Yay for `.sample`!

**Pull request merged**